src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -88,7 +88,7 @@ public static void Initialize(Win32PlatformOptions options)
                 .Bind<IPlatformSettings>().ToConstant(s_instance)
                 .Bind<IPlatformThreadingInterface>().ToConstant(s_instance)
                 .Bind<IRenderLoop>().ToConstant(new RenderLoop())
-                .Bind<IRenderTimer>().ToConstant(new DefaultRenderTimer(60))
+                .Bind<IRenderTimer>().ToConstant(new UiThreadRenderTimer(60))
```

Comment:
Are you sure that you want to switch the render time as a part of this PR?

Reply:
temp to make it work. we need locks prob... any ideas where?